notificationbox/implementation.js

```diff
@@ -59,7 +59,10 @@ class Notification {
       }
     };
 
-    this.getNotificationBox().appendNotification(options.label, notificationId, imageURL, options.priority, buttons, callback);
+    let element = this.getNotificationBox().appendNotification(options.label, notificationId, imageURL, options.priority, buttons, callback);
+    if (options.css) {
+      element.setAttribute("style", element.getAttribute("style") + options.css);
```

Comment:
Even though this works, I think it would be safer to add a semicolor delimiter and space before and after `options.css`. You'd probably end up with double semicolors, but better than none, I'm thinking.